### PR TITLE
WebXR createProjectionLayer: fix options listing

### PR DESCRIPTION
**Pull request by Elchi3**
Definition lists don't really work with nested lists, falling back to just lists.

### CODE REVIEW

File: files/en-us/web/api/xrwebglbinding/createprojectionlayer/index.md [hunk 1]
```diff
@@ -46,17 +44,15 @@ The `options` object can have the following optional properties:
     - `gl.SRGB8`
     - `gl.RGB8_ALPHA8`
    The default value is `gl.RGBA`.
-- `colorFormat` {{optional_inline}}
-  - : A {{domxref("GLenum")}} defining the data type of the depth texture data or `0` indicating that the layer should not provide a depth texture (in that case {{domxref("XRProjectionLayer.ignoreDepthValues")}} will be `true`).
-  Possible values within {{domxref("WebGLRenderingContext")}} contexts with the {{domxref("WEBGL_depth_texture")}} or within {{domxref("WebGLRenderingContext")}} contexts:
+- `depthFormat` {{optional_inline}}: A {{domxref("GLenum")}} defining the data type of the depth texture data or `0` indicating that the layer should not provide a depth texture (in that case {{domxref("XRProjectionLayer.ignoreDepthValues")}} will be `true`).
```
[the remark this answers is not in the document]
Comment (jpmedley) [reply files/en-us/web/api/xrwebglbinding/createprojectionlayer/index.md:47]: If the parenthetical is a complete sentence, it's punctuated as such. If it's a subordinate clause it's punctuated as you did here.

**Review by jpmedley — CHANGES_REQUESTED** [review]

**Review by jpmedley — APPROVED** [review]

**Comment by swanprincess386** [comment]
> Definition lists don't really work with nested lists, falling back to just lists.

**Comment by wbamberg** [comment]
> Definition lists don't really work with nested lists, falling back to just lists.

Definition lists should certainly work with nested lists: if they don't it's a markup problem or a Yari problem. In this case it looks like the alignment of the `dd` contents was a little off, which confused m2h. Lines in the `dd` need to align with the `:`.

The source for this was:

```
- `textureType` {{optional_inline}}
  - : An string defining the type of texture the layer will have. Possible values:
    - `texture`: The textures of {{domxref("XRWebGLSubImage")}} will be of type `gl.TEXTURE_2D`.
    - `texture-array`: the textures of {{domxref("XRWebGLSubImage")}} will be of type `gl.TEXTURE_2D_ARRAY` (WebGL 2 contexts only).
    The default value is `texture`.
- `colorFormat` {{optional_inline}}
  - : A {{domxref("GLenum")}} defining the data type of the color texture data. Possible values:
    - `gl.RGBA`
    - `gl.RGBA`
    Additionally, for contexts with the {{domxref("EXT_sRGB")}} extension enabled:
    - `ext.SRGB_EXT`
    - `ext.SRGB_ALPHA_EXT`
    Additionally, for {{domxref("WebGL2RenderingContext")}} contexts:
    - `gl.RGBA8`
    - `gl.RGB8`
    - `gl.SRGB8`
    - `gl.RGB8_ALPHA8`
   The default value is `gl.RGBA`.
- `colorFormat` {{optional_inline}}
  - : A {{domxref("GLenum")}} defining the data type of the depth texture data or `0` indicating that the layer should not provide a depth texture (in that case {{domxref("XRProjectionLayer.ignoreDepthValues")}} will be `true`).
  Possible values within {{domxref("WebGLRenderingContext")}} contexts with the {{domxref("WEBGL_depth_texture")}} or within {{domxref("WebGLRenderingContext")}} contexts:
    - `gl.DEPTH_COMPONENT`
    - `gl.DEPTH_STENCIL`
    Additionally, for {{domxref("WebGL2RenderingContext")}} contexts:
    - `gl.DEPTH_COMPONENT24`
    - `gl.DEPTH24_STENCIL24`
   The default value is `gl.DEPTH_COMPONENT`.
- `scaleFactor` {{optional_inline}}
  - : A floating-point value which is used to scale the layer during compositing. A value of `1.0` represents the default pixel size for the frame buffer. See also {{domxref("XRWebGLLayer.getNativeFramebufferScaleFactor()")}}. Unlike other layers, the `XRProjectionLayer` can't be created with an explicit pixel width and height, because the size is inferred by the hardware (projection layers fill the observer's entire view).
```

Both lines starting "The default value is " need to be indented by 1 more space, and the line starting "Possible values within" needs to be indented by 2 more spaces. With those changes I get:

Screen Shot 2021-09-21 at 11 43 30 AM

**Comment by Elchi3** [comment]
ahh thanks Will! I should have tried harder. Good to know that this works.

**Pull request merged**